### PR TITLE
Sync packaging scripts with Ubuntu Focal package

### PR DESCRIPTION
**Pull request by nicolasbock**
Signed-off-by: Nicolas Bock <nicolas.bock@canonical.com>

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@nicolasbock Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-server-release&pullRequestId=127)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-server-release&pullRequestId=127) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: packaging/debs/Debian/debian/rabbitmq-server.service [hunk 1]
```diff
@@ -11,29 +10,14 @@ Group=rabbitmq
 UMask=0027
 NotifyAccess=all
 TimeoutStartSec=3600
-
```
Comment (lukebakken) [inline packaging/debs/Debian/debian/rabbitmq-server.service]: Please do not remove commented-out lines.

**Review by lukebakken — CHANGES_REQUESTED** [review]
What is motivating this pull request? There is no information in the PR description.

**Comment by nicolasbock** [comment]
It's motivated by the fact that the files here are outdated compared to Ubuntu's package :smile:

File: packaging/debs/Debian/debian/rabbitmq-server.service (hunk 1 above)
Comment (nicolasbock) [reply packaging/debs/Debian/debian/rabbitmq-server.service]: Done

File: packaging/debs/Debian/debian/rabbitmq-server.init [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
-
 ### BEGIN INIT INFO
 # Provides:          rabbitmq-server
 # Required-Start:    $remote_fs $network
 # Required-Stop:     $remote_fs $network
 # Default-Start: 2 3 4 5
 # Default-Stop: 0 1 6
 # Description:       RabbitMQ broker
-# Short-Description: Manages RabbitMQ server
+# Short-Description: Enable AMQP service provided by RabbitMQ broker
```
Comment (michaelklishin) [inline packaging/debs/Debian/debian/rabbitmq-server.init:9]: Can we please keep the description our team wants to see?

**Comment by michaelklishin** [comment]
@nicolasbock I assume you do realise this repository is the canonical [no pun intended] source for [our Debian packages](https://www.rabbitmq.com/install-debian.html). We fully respect the Debian  community's approach to packaging but the sad fact is that the packages in Debian and Ubuntu standard repos are always at least 6 (more like 12) behind RabbitMQ releases. Some very recent examples:

 * Eoan ships RabbitMQ 3.7.8 from [September 2018](https://www.rabbitmq.com/changelog.html)
 * Focal ships 3.8.2 from December 2019

So in what areas are we supposed to be catching up with Ubuntu?

**Comment by michaelklishin** [comment]
We will not merge PRs where the motivation is "sync X from Y because it's out of date with Y". This package is **not a downstream** from the one that the Debian folks maintain.

If you want us to accept some changes the broader Debian community wants to see, we would consider them but please take some time to explain them.

I would also very much like that our package's description and other fields were not modified to be those in Ubuntu. RabbitMQ is not limited to "an AMQP service" and "AMQP" these days is two separate protocols that historically happen to share a portion of their name. RabbitMQ supports both but we prefer to be specific.

**Comment by michaelklishin** [comment]
I see just two meaningful changes in this PR:

 * `rabbitmqctl wait` is passed a timeout taken from `RABBITMQ_TIMEOUT`. Perhaps a decent idea but a very vague variable name.
 * Log rotation uses a suffix defined using an environment variable.

Everything else is cosmetic, useful link removal or uncalled for copy edits. In case there's interest in improving on either of those fronts, we'd consider more specific PRs.

**Comment by nicolasbock** [comment]
Thanks @michaelklishin for the detailed feedback. I have created #129 which is hopefully more focused and acceptable for inclusion into this repository. Thanks!

**Pull request closed without merging**